<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clock_in_out</class>
 <widget class="QWidget" name="Clock_in_out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;welcome！name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;上班状态：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;本次已上班时长：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;本日已上班时长：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
        </property>
        <property name="text">
         <string>签到</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
        </property>
        <property name="text">
         <string>本日下班</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>492</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
